src/XacroParser.js

```diff
@@ -214,7 +214,11 @@ export class XacroParser {
                 }
 
                 obj.name = name;
-                obj.def = def;
+                if (def.startsWith('\'') && def.endsWith('\'')) {
+                    obj.def = def.substr(1, def.length - 2);
```

Comment:
Can we use "substring" here, instead. "substr" is listed as legacy in the [mdn docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#the_difference_between_substring_and_substr).